### ISSUE TITLE
Invalid type for block.transactions

### ISSUE THREAD

**Issue by sharifzadesina** [tronprotocol/tronweb#522]
When a block doesn't have any transactions, `block.transaction` property won't be present and it will be undefined.

but the type for `block.transaction` is an array `TronTransaction<ContractParamter>[]` it should be `TronTransaction<ContractParamter>[] | undefined` or make it return an empty array.

This is a simple code that retrives an empty block:

```ts
import { AxiosHeaders } from "axios";
import { TronWeb } from "tronweb";

const tw = new TronWeb({
  fullHost: 'https://api.shasta.trongrid.io',
  headers: new AxiosHeaders({ 'TRON-PRO-API-KEY': 'x-x-x-x-x' })
});

// const block = await tw.trx.getBlock(44711520); // empty block
const block = await tw.trx.getBlock(33273991); // non-empty block

console.log(block);
```

**Comment by start940315** [comment]
Thanks for your issue. We'll fix it in our next version.

**Comment by start940315** [comment]
We just published `TronWeb@v6.0.0-beta.4` which fixed this issue. Welcome to try:
```bash
npm install tronweb@beta
```
